doc/source/guide/acquiring_data/fido.rst

```diff
@@ -0,0 +1,68 @@
+-----------------------------
+Downloading Data using Fido
+-----------------------------
+
+This guide outlines how to search for and download data using SunPy's
+Federated Internet Data Obtainer...or more usually (and
+sanely) referred to as Fido.  Fido is a unified API for seeking
```

Comment:
Good question, I prefer if it was FIDO but its not how its called in the code.
So I think we don't want to confuse users by going FIDO and then having to then use Fido.attrs.